config/.markdownlint.yaml

```
@@ -0,0 +1,200 @@
+# Rule details can be found at https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md
+
+default: false
+
+# MD001/heading-increment/header-increment - Heading levels should only increment by one level at a time
+MD001: true
+# MD002/first-heading-h1/first-header-h1 - First heading should be a top-level heading (deprecated)
+MD002: false
+
+# MD003/heading-style/header-style - Heading style
+MD003:
+  # Heading style
+  style: "atx"
+
+# MD004/ul-style - Unordered list style
+MD004:
+  # List style
+  style: "consistent"
+
+# MD005/list-indent - Inconsistent indentation for list items at the same level
+MD005: true
+
+# MD006/ul-start-left - Consider starting bulleted lists at the beginning of the line (deprecated)
+MD006: false
+# MD007/ul-indent - Unordered list indentation
+MD007: false
+# MD009/no-trailing-spaces - Trailing spaces
+MD009:
+  # Spaces for line break
+  br_spaces: 2
+  # Allow spaces for empty lines in list items
+  list_item_empty_lines: false
+  # Include unnecessary breaks
+  strict: false
+
+# MD010/no-hard-tabs - Hard tabs
+MD010: false
+
+# MD011/no-reversed-links - Reversed link syntax
+MD011: true
+
+# MD012/no-multiple-blanks - Multiple consecutive blank lines
+MD012: false
+
+# MD013/line-length - Line length
+# Changed from default due to the fact that paragraphs should be on a single line.
+MD013: false
```

```suggestion
MD013: true
```
I feel pretty strongly in my objection to requiring paragraphs be a single line. @Pandapip1  it doesn't bring benefit (other than on any individual person's IDE with "soft wrap on"), it hurts the readability and ability to pin-point

---

Why do you suggest enabling this rule, then>

---

Doesn't `false` means there will _not_ be limiting length?

---

`false` means the rule is disabled - i.e. the lint will not be applied.

---

`true` means the lint is enabled - i.e. there *will* be a max length.

---

The default usually involves a max length of 80, 100 or 120 and most Markdown Linter  follows it. I am fine to leave the linting `MD013: false`, to not enforce a max length, but I wonder if you flip it to say "should be on a single line", that's what I vote no for,

---

LGTM on keeping it `false` now that wording changed